### DISCUSSION TITLE
Does keygen support redis-sentinel?

### DISCUSSION THREAD

**Discussion by jceddy** [keygen-sh/community#156] [Q&A] [answered]
All of the example configurations I can find set REDIS_URL to point to a single host. If we are connecting to cluster, can REDIS_URL be configured to use a sentinel URL?

I need to know whether I can use sentinel (preferred), or if I need to come up with some kind of load-balancer kludge to have it dynamically route to the master redis node.

**Comment by ezekg** [comment, marked as answer]
I don't think Keygen works with Redis Sentinel out-of-the-box, but open to a PR if you want to [implement](https://github.com/redis/redis-rb?tab=readme-ov-file#sentinel-support) in a backwards compatible way.

Prior art: https://github.com/search?type=code&q=sentinels%3A+path%3Aconfig%2F**%2F*.rb
